LICENSE

```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Sanctuary
```

Comment:
Should we include the following as well?

    Copyright (c) 2015 Simon Friis Vindum

If you just drew inspiration from [union-type][1] this isn't necessary, but if you used its source code as a starting point I believe Simon has copyright. Fortunately union-type is MIT-licensed. :)

/cc @paldepind


[1]: https://github.com/paldepind/union-type